charts/incubator/plausible/questions.yaml

```diff
@@ -134,13 +126,78 @@ questions:
               - value: Bamboo.PostmarkAdapter
                 description: Bamboo.PostmarkAdapter
+              - value: Bamboo.MailgunAdapter
+                description: Bamboo.MailgunAdapter
+              - value: Bamboo.MandrillAdapter
+                description: Bamboo.MandrillAdapter
+              - value: Bamboo.SendGridAdapter
+                description: Bamboo.SendGridAdapter
         - variable: POSTMARK_API_KEY
           label: POSTMARK_API_KEY
-          description: Enter your API key.
+          description: Required. Enter your API key.
+          schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.PostmarkAdapter"]]
+            type: string
+            default: ""
+            private: true
+        - variable: MAILGUN_API_KEY
+          label: MAILGUN_API_KEY
+          description: Required. Enter your API key.
           schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.MailgunAdapter"]]
             type: string
             default: ""
             private: true
+        - variable: MAILGUN_DOMAIN
+          label: MAILGUN_DOMAIN
+          description: Required. Enter your Mailgun domain.
+          schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.MailgunAdapter"]]
+            type: string
+            default: ""
+        - variable: MAILGUN_BASE_URI
+          label: MAILGUN_BASE_URI
+          description: This is optional. Mailgun makes a difference in the API base URL between sender domains from within the EU and outside. By default, the base URL is set to https://api.mailgun.net/v3. To override this you can pass https://api.eu.mailgun.net/v3 if you are using an EU domain.
+          schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.MailgunAdapter"]]
+            type: string
+            default: ""
+        - variable: MANDRILL_API_KEY
+          label: MANDRILL_API_KEY
+          description: Required. Enter your API key.
+          schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.MandrillAdapter"]]
+            type: string
+            default: ""
+            private: true
+        - variable: SENDGRID_API_KEY
+          label: SENDGRID_API_KEY
+          description: Required. Enter your API key.
+          schema:
+            show_if: [["MAILER_ADAPTER", "=", "Bamboo.SendGridAdapter"]]
+            type: string
+            default: ""
+            private: true
+        - variable: MAXMIND_LICENSE_KEY
+          label: MAXMIND_LICENSE_KEY
+          description: MaxMind license key to automatically download and update the database
+          schema:
+            type: string
+            default: ""
+            private: true
+        - variable: MAXMIND_EDITION
+          label: MAXMIND_EDITION
+          description: MaxMind database edition to use (only if MAXMIND_LICENSE_KEY is set)
+          schema:
+            show_if: [["MAXMIND_LICENSE_KEY", "!=", ""]]
+            type: string
+            required: true
```

Comment:
Honestly not sure if this works together with `show_if`. Should I just remove `required: true`?

Reply:
It does work! It will only be required if the licence key is not empty!